bin/doctrine-migrations.php

```diff
@@ -43,7 +47,7 @@
 $helperSet = ($helperSet) ?: new \Symfony\Component\Console\Helper\HelperSet();
 $helperSet->set(new \Symfony\Component\Console\Helper\DialogHelper(), 'dialog');
 
-$cli = new \Symfony\Component\Console\Application('Doctrine Migrations', \Doctrine\DBAL\Migrations\MigrationsVersion::VERSION);
+$cli = new \Symfony\Component\Console\Application('Doctrine Migrations', '@git-version@');
```

Comment:
This only works for the PHAR package, right? I assume this will break, if I use this library as dependency.